core/thegraph/config.go

```diff
@@ -39,15 +41,22 @@ func CLIFlags(envPrefix string) []cli.Flag {
 			Value:  5,
 			EnvVar: common.PrefixEnvVar(envPrefix, "GRAPH_MAX_RETRIES"),
 		},
+		cli.IntFlag{
+			Name:   OperatorStateCacheSize,
+			Usage:  "The size of the operator state cache in elements (0 to disable)",
+			Value:  0,
```

Comment:
Should default be 0 or 32?

Reply:
What's the estimated size bytes for 32 entries and how many instances of this struct is expected to be created? If it's small enough it should be fine to enable by default

Reply:
Depends on the amount of operators but let's assume 200. The order of magnitude is ~1-10mb.

Reply:
Actually, it seems this cache is not really useful:
- At different components, it different instances been created so they don't share the cache
- Within same components, like controller, it may not need to access same blockNumber many times, only when it creates batch (https://github.com/Layr-Labs/eigenda/blob/master/disperser/controller/dispatcher.go#L294), and then the blockNumber will advance

Where does the cache help?